gulpfile.js

```diff
@@ -0,0 +1,23 @@
+'use strict';
+
+const gulp = require('gulp');
+const eslint = require('gulp-eslint');
+const mocha = require('gulp-mocha');
+
+gulp.task('test', function(){
+  gulp.src('./test/*-test.js', {read: false})
+  .pipe(mocha({reporter:'nyan'}));
+});
+
+gulp.task('lint', function(){
+  return gulp.src(['**/*.js', '!node_modules/**'])
+    .pipe(eslint())
+    .pipe(eslint.format())
+    .pipe(eslint.failAfterError());
+});
+
+gulp.task('dev', function(){
+  gulp.watch(['**/*.js', '!node_modules/**'], ['lint', 'test']);
+});
+
+gulp.task('default', ['dev']);
```

Comment:
@brials @remilonwheels @caylazabel `gulpfile.js` looks good - be sure to add additional tasks, as needed, for your own personal workflow